<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SaveSettingsTool</class>
 <widget class="QWidget" name="SaveSettingsTool" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>75</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="2" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>85</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QCheckBox" name="chkProgressiveLoading" >
     <property name="text" >
      <string>Progressive Loading</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QCheckBox" name="chkShowPreview" >
     <property name="text" >
      <string>Show Preview</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Quality&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSlider" name="sldQuality" >
     <property name="maximum" >
      <number>100</number>
     </property>
     <property name="value" >
      <number>85</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QComboBox" name="cmbType" >
     <item>
      <property name="text" >
       <string>JPEG -- lower-quality/smaller files</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>PNG -- perfect-quality/large files</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4" >
    <widget class="QLabel" name="lblSize" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Image Size: &lt;span style=&quot; font-style:italic;&quot;>102k&lt;/span> - Loading Time: Modem &lt;span style=&quot; font-style:italic;&quot;>26 seconds&lt;/span>, Fast Connection &lt;span style=&quot; font-style:italic;&quot;>2 seconds&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="0" column="4" >
    <widget class="QPushButton" name="btnClose" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>1</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>sldQuality</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>429</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel" >
     <x>543</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>SaveSettingsTool</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>735</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel" >
     <x>710</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
